package.json

```diff
@@ -21,7 +21,9 @@
     "lint": "standard ./lib/**/*.js",
     "tape": "tape ./tests/**/*.test.js | tap-spec",
     "nyc": "nyc --reporter=text --reporter=text-summary --reporter=html --report-dir=docs/reports/coverage npm run tape",
-    "postnyc": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100"
+    "postnyc": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
+    "semantic-release": "semantic-release",
```

Comment:
I don't think you need the semantic release script. `npx sematic-release` should find the module and execute it.